.drone.yml

```diff
@@ -1,27 +1,35 @@
 build:
   image: golang:1.5
-  environment:
-    - GO15VENDOREXPERIMENT=1
-    - GOOS=linux
-    - GOARCH=amd64
-    - CGO_ENABLED=0
   commands:
-    - go get
-    - go build
-    - go test
+    - make deps
```

Comment:
We already did that before for a lot other plugins, now I have written a more or less unified makefile that is/will be part of all official plugins. I think especially while developing it's nicer to execute `make deps vet build test` instead of executing all commands separately.